install/base-cluster-roles.yaml

```diff
@@ -190,19 +190,37 @@ rules:
       - update
 ---
 
-## WIll be removed and replaced with RoleBinding for specific projects. 
-kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
 metadata:
-  name: runai-scheduler-rw
+  namespace: runai
+  name: runai-scheduler-runai-ns-only
```

Comment:
why change the name?  (the fact that it is a Role (and not a ClusterRole) means it's only for a single namespace)